ofl/eduqldhandprecursive/METADATA.pb

```diff
@@ -0,0 +1,42 @@
+name: "Edu QLD Hand Precursive"
+designer: "Tina Anderson, Corey Anderson"
+license: "OFL"
+category: "HANDWRITING"
+date_added: "2024-03-19"
+fonts {
+  name: "Edu QLD Hand Precursive"
+  style: "normal"
+  weight: 400
+  filename: "EduQLDHandPrecursive[wght].ttf"
+  post_script_name: "EduQLDHandPrecursive-Regular"
+  full_name: "Edu QLD Hand Precursive Regular"
+  copyright: "Copyright 2023 The QLD School Hand Australia Project Authors (https://github.com/SorkinType/QLDSchoolHandAustralia)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "vietnamese"
+axes {
+  tag: "wght"
+  min_value: 400.0
+  max_value: 700.0
+}
+source {
+  repository_url: "https://github.com/SorkinType/QLDSchoolHandAustralia"
+  commit: "d39073e405921a64f2078aa9e6a31380bc9f5e55"
+  archive_url: "https://github.com/SorkinType/QLDSchoolHandAustralia/releases/download/v1.000/QLDSchoolHandAustralia-v1.000.zip"
+  files {
+    source_file: "OFL.txt"
+    dest_file: "OFL.txt"
+  }
+  files {
+    source_file: "fonts/variable/EduQLDHandPrecursive[wght].ttf"
+    dest_file: "EduQLDHandPrecursive[wght].ttf"
+  }
+  files {
+    source_file: "DESCRIPTION.en_us.html"
+    dest_file: "DESCRIPTION.en_us.html"
+  }
+  branch: "main"
+}
+display_name: "Edu QLD Hand Precursivecursive"
```

Comment:
> Precursivecursive
I think there is a typo here, but I rather ask you to confirm about it

Reply:
Yes, it should either be Precursive or cursive

Reply:
Thanks. Changed to "Precursive".